job-runner/main.go

```diff
@@ -14,9 +14,21 @@ import (
 )
 
 func main() {
+	// Make a chain repo.
+	chainRepo := chain.NewMemoryRepo()
```

Comment:
For the sake of this PR, we can punt this and do another PR later where we introduce a config file so we and users can specify a Redis `addr:port` and if specified then use Redis, else fallback to memory.